code/modules/unit_tests/plane_dupe_detector.dm

```diff
@@ -0,0 +1,12 @@
+///Checks if some idiot didnt give two planes the same layer
+/datum/unit_test/plane_dupe_detector
+
+/datum/unit_test/plane_dupe_detector/Run()
+	var/list/plane_integer_list = list()
+	for(var/atom/movable/screen/plane_master/air as anything in subtypesof(/atom/movable/screen/plane_master))
```

Comment:
air? just name this plane_master_path or something

Reply:
Yeah then it spells air.plane

Reply:
I think that’s pretty cool

Reply:
Bro

Reply:
I’m really sad you don’t like it, but fine I’ll change it in a bit

Reply:
death by a thousand paper cuts/silly names